src/cloud/docker/docker-containers-cli.md

```diff
@@ -0,0 +1,101 @@
+---
+group: cloud-guide
+title: Docker CLI Containers
+functional_areas:
+  - Cloud
+  - Docker
+  - Configuration
+---
+
+The following CLI containers, most of which are based on a [PHP-CLI version 7 image], provide `magento-cloud` and `{{site.data.var.ct}}` commands to perform file system operations and interact with the application:
+
+-  `build`—extends the CLI container to perform operations with writable filesystem, similar to the build phase
+-  `deploy`—extends the CLI container to use read-only file system, similar to the deploy phase
+-  `cron`—extends the CLI container to run cron
+-  `node`—based on node image, used for NPM activities
+
+For example, you can check the state of the your project using the _ideal-state_ wizard:
+
+Run the `{{site.data.var.ct}}` ideal-state command.
+
+```bash
+docker-compose run deploy ece-command wizard:ideal-state
+```
+
+Sample response:
+
+```terminal
+ - Your application does not have the "post_deploy" hook enabled.
+The configured state is not ideal
+```
+{:.no-copy}
+
+All build and deploy processes are defined and configured using {{site.data.var.ct}} and the Magento Cloud template.
+
+## CLI container commands
+
+These commands are available in the Magento Cloud Docker environment:
+
+| Command    | Target Containers   |  Notes
+| ------------- |  ------------------ |------------------
+| `cloud-build` | build | Used to build the application in production mode, configured via build hook in .magento.app.yml
+| `cloud-deploy` | deploy | Used to deploy the application, configured via deploy hook in .magento.app.yml
+| `cloud-post-deploy` | deploy | Used to deploy the application, configured via deploy hook in .magento.app.yml
+| `ece-command` | deploy | Used to run other commands from ece-tools CLI Tool
+| `magento-command` | deploy | Used to run bin/magento commands
+| `magento-installer` | deploy | Just runs build and then deploy hooks
```

Comment:
I am not aware of this command. Does it run post-deploy too? We had `redeploy` command before but this one looks new. @shiftedreality please validate.

Reply:
Yes, `magento-installer` is deprecated and exists only for BC - please remove